app/src/main/java/org/sopt/sopt_collaboration_panelnow/presentation/home/screen/HomeScreen.kt

```diff
@@ -0,0 +1,71 @@
+package org.sopt.sopt_collaboration_panelnow.presentation.home.screen
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.sopt.sopt_collaboration_panelnow.presentation.home.viewmodel.HomeViewModel
+import org.sopt.sopt_collaboration_panelnow.presentation.home.component.MiniTestSection
+import org.sopt.sopt_collaboration_panelnow.presentation.home.component.PointTransactionsSection
+import org.sopt.sopt_collaboration_panelnow.presentation.home.component.PopularSurveySection
+import org.sopt.sopt_collaboration_panelnow.presentation.home.model.MiniTestModel
+
+@Composable
+fun HomeScreen(
+    miniTests: List<MiniTestModel>,
+    modifier: Modifier = Modifier,
+    viewModel: HomeViewModel = hiltViewModel(),
+) {
+    val scrollState = rememberScrollState()
+    val homeUiState by viewModel.homeUiState.collectAsState()
```

Comment:
```suggestion
    val homeUiState by viewModel.homeUiState.collectAsStateWithLifeCycle()
```
이번 세미나 때  Flow 에 대해서 배웠었죠! 데이터 수집과 일관성을 위해 아래와 같이 바꿔주는게 좋아보입니다